core/client/app/styles/patterns/global.css

```diff
@@ -6,14 +6,24 @@
 /* ---------------------------------------------------------- */
 
 :root {
+    /* Colours' RGB */
+    --rgb-darkgrey: 36,38,40;
+    --rgb-midgrey: 125,135,138;
+    --rgb-lightgrey: 223,225,227;
+    --rgb-superlightgrey: 226,237,242;
+    --rgb-blue: 91,164,229;
+    --rgb-red: 226,84,64;
+    --rgb-orange: 242,169,37;
+    --rgb-green: 159,187,88;
     /* Colours */
-    --darkgrey: #242628;
-    --midgrey: #7d878a;
-    --lightgrey: #e2edf2;
-    --blue: #5ba4e5;
-    --red: #e25440;
-    --orange: #f2a925;
-    --green: #9fbb58;
+    --darkgrey: rgb(var(--rgb-darkgrey));
+    --midgrey: rgb(var(--rgb-midgrey));
+    --lightgrey: rgb(var(--rgb-lightgrey));
+    --superlightgrey: rgb(--rgb-superlightgrey);
```

Comment:
I'm not sure about `superlightgray` as a name for this colour, feel free to suggest something better.